src/components/UserInfo.js

```diff
@@ -0,0 +1,33 @@
+export default class UserInfo {
+  constructor({ titleSelector, descriptionSelector, avatarSelector }) {
+    this._title = document.querySelector(titleSelector);
+    this._description = document.querySelector(descriptionSelector);
+    this._avatar = document.querySelector(avatarSelector);
+  }
+
+  //new phrasing below 6.5
+  getUserInfo() {
+    return {
+      title: this._title.textContent,
+      description: this._description.textContent,
+    };
+  }
+
+  setUserInfo({ title, description, avatar }) {
+    this._title.textContent = title;
+    this._description.textContent = description;
+    this._avatar.src = avatar;
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```

Reply:
Still needs correction